src/popup/components/Modals/SecureLogin.vue

```diff
@@ -117,7 +117,7 @@ export default defineComponent({
     margin-top: 8px;
 
     .title {
-      @extend %face-sans-18-semi-bold;
+      @extend %face-sans-18-medium;
```

Comment:
`semi-bold` had a `font-weight` of 600 while `medium` has 500
In the design the title has a `font-weight` of 600. 
I don't understand why `face-sans-18-semi-bold` was removed